RELEASE_NOTES.md

```diff
@@ -1,5 +1,16 @@
 ﻿RELEASE NOTES
 
+**Version**: 1.1
+
+This versions requires:
+* Shotgun for Unity 1.1 (com.unity.integrations.shotgun)
```

Comment:
Is this the correct version? usually unity packages versions are like x.y.z[-(pre|exp|rc).w]